Package.swift

```diff
@@ -30,10 +30,5 @@ let package = Package(
                         .copy("app/img"),
                         .copy ("app/public"),
                         .copy ("app/index.html")]),
-
```

Comment:
Alex asked to remove this as it‘s empty and conflicting with the BSK target with the same name